docs/New Playbooks Contribution Guide/readme.md

```diff
@@ -235,7 +235,7 @@ Fields marked with * are mandatory. Others can be left empty.
         "lastUpdateTime": "2021-05-18T10:00:15.123Z", 
         "entities": ["Account"], 
         "tags": ["Identity protection", "Teams bot"], 
-        "source": {
+        "support": {
```

Comment:
We also have email/link/name if you want to add that
```
  "support": {
    "name": "Microsoft",
    "email": "support@microsoft.com",
    "link": "https://support.microsoft.com/",
    "tier": "developer"
  },
```

Reply:
Thanks!
"tier" will replace "name"
we will add "link" in case this is not Microsoft or Community (in these cases, gallery will automatically show the relevant link to Microsoft support / GitHub issue